src/taipy/core/_entity/_migrate.py

```diff
@@ -113,18 +107,27 @@ def _migrate_scenario(scenario: Dict, data: Dict) -> Dict:
 
 
 def __is_cacheable(task: Dict, data: Dict) -> bool:
-    for id in task["output_ids"]:
-        dn = data.get(id, {})
-        if "cacheable" not in dn or not dn["cacheable"]:
+    output_ids = task.get("output_ids", []) or task.get("outputs", [])  # output_ids is on entity, outputs is on config
+
+    for output_id in output_ids:
+        if output_id.endswith(":SECTION"):  # Get the config_id if the task is a Config
+            output_id = output_id.split(":")[0]
+        dn = data.get(output_id, {})
+        if "data" in dn:
+            dn = dn.get("data", {})
+
+        if "cacheable" not in dn or not dn["cacheable"] or dn["cacheable"] == "False:bool":
             return False
     return True
 
 
 def _migrate_task(task: Dict, data: Dict) -> Dict:
-    # owner_id was not present in 2.0
-    if task.get("parent_id"):
-        task["owner_id"] = task.get("owner_id")
+    # parent_id has been renamed to owner_id
+    try:
```

Comment:
I would maintain the original structure in this case, just fixing the worng key on line 126. Catching a exception just to do nothing adds unnecessary complexity. But is just preference, feel free to keep it as you like.

Reply:
I changed it because in case `"parent_id': null`, the `if task.get("parent_id")` also is None, so the `"owner_id"` will not exist in the entity, which is not what we want.